packages/core/src/core.ts

```diff
@@ -100,6 +100,25 @@ export function getInput(name: string, options?: InputOptions): string {
   return val.trim()
 }
 
+/**
+ * Gets the values of an multiline input.  Each value is also trimmed.
+ *
+ * @param     name     name of the input to get
+ * @param     options  optional. See InputOptions.
+ * @returns   string[]
+ *
+ */
+export function getMultilineInput(
+  name: string,
+  options?: InputOptions
+): string[] {
+  const inputs: string[] = getInput(name, options)
+    .split('\n')
```

Comment:
should we always split on `\n` or should we support `os.EOL`? I'm just curious if the current implementation might confused windows users?

Reply:
It needs to be \n because that's how these get mapped into env variables in the runner! \r\n won't work!